Channels/Insider/release_notes.md

```diff
@@ -2,6 +2,10 @@
 
 Welcome to the April 18th, 2019 release of Accessibility Insights for Windows.
```

Comment:
The date in each release notes file is the date that it was deployed to that specific ring